### ISSUE TITLE
False positive on screenshots success

### ISSUE THREAD

**Issue by x1mdev** [michenriksen/aquatone#125]
Hey,

I've ran a couple of test runs on a RPi3+ with a Kali OS running on it and I seem to get a false positive on the screenshot success message.

This is the output after a successful run: 

<img width="1069" alt="screenshot 2018-11-05 at 16 00 26" src="https://user-images.githubusercontent.com/22507577/48005565-0b4d2200-e114-11e8-9bb0-9749c8662153.png">

Running `Linux kali 4.14.62-Re4son-v7+ #2 SMP Sat Sep 22 22:53:40 AEST 2018 armv7l GNU/Linux` with chromium. It may be due to the error from chromium that tells us to use `--no-sandbox`, but I saw something in the code about that here: [url_screenshotter.go](https://github.com/michenriksen/aquatone/blob/master/agents/url_screenshotter.go)

**Comment by michenriksen** [comment]
That's interesting. I do have code to add the `--no-sandbox` flag in case the EUID of the user is zero. I'll see if I can reproduce.

**Comment by x1mdev** [comment]
Yeah that's what I saw too, I'll run a few more tests when I get home.

**Comment by shadowzoom** [comment]
> That's interesting. I do have code to add the `--no-sandbox` flag in case the EUID of the user is zero. I'll see if I can reproduce.

Guys did you figure it out? Because i don't have screenshots too :(
My command is 
cat targets.txt | ./aquatone -chrome-path "/mnt/c/Users/Admin/desktop/tools/aquatone_windows_amd64_1.4.2/chrome-win" -debug

**Comment by jdonsec** [comment]
I'm currently getting this error and no screenshots on Ubunut 18.04
cat target.txt | aquatone -chrome-path /snap/bin

error is:
screenshot failed: fork/exec /snap/bin: permission denied
